mep/footnotes/templates/footnotes/snippets/card_results.html

```diff
@@ -0,0 +1,29 @@
+{% load static %}
+<style nonce="{{ request.csp_nonce }}">
```

Comment:
I'm OK with removing this since the numbers aren't getting read anyway, and we aren't showing them to sighted users.